ci/test/00_setup_env_mac.sh

```diff
@@ -7,7 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export HOST=x86_64-apple-darwin14
-export PACKAGES="cmake imagemagick libcap-dev librsvg2-bin libz-dev libbz2-dev libtiff-tools python3-dev python3-setuptools"
+export BREW_PACKAGES="automake berkeley-db4 libtool boost miniupnpc openssl pkg-config protobuf qt qrencode librsvg pyenv"
```

Comment:
why is this changed?

Reply:
By accident. Fixed. This PR is still very rough; I can pick it up again now that #13728 is merge.